blips/blip-2-researchcertificate.md

```diff
@@ -0,0 +1,112 @@
+
+```json
+{
+    "title": "Research Object Metadata",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string",
+            "required": false,
+            "description": "Identifies the asset to which this NFT represents"
+        },
+        "description": {
+            "type": "string",
+            "required": false,
+            "description": "Describes the asset to which this NFT represents"
+        },
+        "image": {
+            "type": "string",
+            "required": false,
+            "description": "A URI pointing to a resource with mime type image/* representing the asset to which this NFT represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive."
+        },
+        "claimant": {
+            "author(s)": {
+                "type": "array",
+                "required": true,
+                "description": "Array of the contributors and/or authors of the data associated with the corresponding ISCC."
+            },
+            "organization(s)": {
+                "type": "array",
+                "required": false,
+                "description": "Array of organizations that are stakeholders in the certified data."
+            }
+        },
+        "iscc": {
+            "type": "string",
+            "required": true,
+            "description": "ISCC is an algorithmic identifier generated from the content itself. This currently includes metadata similarity, normalized content similarity, encoded data similarity, and exact data integrity."
+        },
+        "certificate": {
+            "type": "string",
+            "required": false,
+            "description": "a URI pointing to the corresponding data certificate in PDF format. The certificate can be utilized to prove via certified timestamping the data certification time."
+        },
+        "external_url": {
+            "type": "string",
+            "required": false,
+            "description": "a URI pointing to the corresponding data file where the data can be downloaded. If provided, the data at the URI should resolve to the same ISCC listed in the metadata."
+        }
+        ,
+        "timestamp": {
+            "type": "string",
+            "required": true,
+            "description": "UNIX timestamp that corresponds to the block confirmation time when the research data object token was minted."
+        }
+    }
+}
+```
+
+The research data schema offers flexibility in what specific metadata fields are required to be included. This is for several reasons: 1. Data generated at differ stages of the research cycle, it may not be possible to provide this information. 2. To ensure the protection of sensitive and confidential data that is acquired, analyzed, or processed. However, the required field ISCC, is required in order to ensure subsequent validation of data at a later point in time.
```

Comment:
Something I learned from using LaTeX in Git was using 1 line per sentence, might be good here also.

Reply:
What is the benefit of this? More readability or formatting?

Reply:
Diffs in Git become very easy to read. Can also make merging simpler.